src/components/App.js

```diff
@@ -47,6 +47,8 @@ import EditorTabs from "./Editor/Tabs";
 
 import SymbolModal from "./SymbolModal";
 
+import GotoLineModal from "./GotoLineModal";
```

Comment:
this whole pattern is from @codehag's diff on the associated issue. 

It feels like these (SymbolModal, ShortcutModal, GotoLineModal) should all be the same thing, but I thought this would be a good start before merging them together.

Reply:
Yeah... we do want to consolidate the symbol + gotoline modal and search for file.

+ `cmd+p`
+ `cmd+:`
+ `cmd+shift+o`

Reply:
- [cmd+p](https://github.com/devtools-html/debugger.html/blob/master/assets/panel/debugger.properties#L141)
- [cmd+shift+o](https://github.com/devtools-html/debugger.html/blob/master/assets/panel/debugger.properties#L153)

For those shortcuts... `CmdOrCtrl+Shift+;` works, but `CmdOrCtrl+:` does not work, nor does `CmdOrCtrl+Shift+:`. Is this correct? This could be revisted when you unify them, whatever you'd like for right now is fine by me :)